<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleRenderWdg</class>
 <widget class="QWidget" name="DoubleRenderWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DoubleRenderWdg</string>
  </property>
  <widget class="VhallRenderMaskWdg" name="mVhallRenderMaskWdg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="CVideoRenderReceiveWdg" name="widget_teacher" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	
	background-color: rgb(85, 255, 127);
}</string>
   </property>
   <widget class="QWidget" name="widget_student" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CVideoRenderReceiveWdg</class>
   <extends>QWidget</extends>
   <header>cvideorenderreceivewdg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VhallRenderMaskWdg</class>
   <extends>QWidget</extends>
   <header>vhallrendermaskwdg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
